docs/sources/get-started/labels/_index.md

```diff
@@ -9,83 +9,282 @@ aliases:
+{{% /admonition %}}
+
+{{< admonition type="tip" >}}
+For Grafana Cloud Logs, see the [current OpenTelemetry guidance](https://grafana.com/docs/grafana-cloud/send-data/otlp/otlp-format-considerations/#logs).
+{{< /admonition >}}
+
+The default list of resource attributes to store as labels can be configured using `default_resource_attributes_as_index_labels` under the [distributor's otlp_config](https://grafana.com/docs/loki/<LOKI_VERSION>/configure/#distributor). You can set global limits using [limits_config.otlp_config](/docs/loki/<LOKI_VERSION>/configure/#limits_config). If you are using Grafana Cloud, contact support to configure this setting.
+
+## Labeling is iterative
+
+You want to start with a small set of labels. While accepting the default labels assigned by Grafana Alloy or the OpenTelemetry Collector or the Kubernetes Monitoring Helm chart may meet your needs, over time you may find that you need to modify your labeling strategy.
+
+Once you understand how your first set of labels works and you understand how to apply and query with those labels, you may find that they don’t meet your query patterns.  You may need to modify or change your labels and test your queries again.
+
+Settling on the right labels for your business needs may require multiple rounds of testing. This should be expected as you continue to tune your Loki environment to meet your business requirements.
+
+## Create low cardinality labels
+
+[Cardinality](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/labels/cardinality/) refers to the combination of unique labels and values which impacts the number of log streams you create.  High cardinality causes Loki to build a huge index and to flush thousands of tiny chunks to the object store. Loki performs very poorly when your labels have high cardinality. If not accounted for, high cardinality will significantly reduce the performance and cost-effectiveness of Loki.
+
+High cardinality can result from using labels with an unbounded or large set of possible values, such as timestamp or ip_address **or** applying too many labels, even if they have a small and finite set of values.
+
+High cardinality can lead to significant performance degradation. Prefer fewer labels, which have bounded values.
+
+## Creating custom labels
+
+{{< admonition type="tip" >}}
+Many log collectors such as Grafana Alloy, or the Kubernetes Monitoring Helm chart, will automatically assign appropriate labels for you, so you don't need to create your own labeling strategy.  For most use cases, you can just accept the default labels.
+{{< /admonition >}}
+
+Usually, labels describe the source of the log, for example:
+
+- the name of the application which produced the message
+- the filename of the source log file on disk
+- the hostname where the log was produced
```

Comment:
I think I would replace these with:
```
- the namespace or additional logical grouping of applications
- cluster, and/or region of where the logs were produced
```

Reply:
if we want to add `hostname` I think it should have a caveat that it be used in situations where people regularly query by hostname such as in environments that have a lot of individually named machines or virtual machines, for environment with ephemeral machines or virtual machines host should be stored in structutured metadata